### ISSUE TITLE
Error launching fortls

### ISSUE THREAD

**Issue by lukhirt** [hansec/vscode-fortran-ls#43]
Hi,

I am getting the following error:

> Error launching fortls: Please check that all selected options are supported by your language server version.

1. I have FLS 1.12.0 installed through pip:
- done the debugging as described on the [FLS WIKI](https://github.com/hansec/fortran-language-server/wiki) and all seems correct.
2. I have python installed (3.8.3).
3. I have Modern Fortran extension installed as well.

Is there any way I could see what exactly is the problem? Is there any kind of log file for this extension? I suppose that since I have latest version of FLS the arguments I am providing should not be a problem, right?

Thanks in advance!

**Comment by lukhirt** [comment]
A bit of an update.
I've played a bit with the code inside extension.js. Looks that even when fortls is called w/o any arguments I get the non-zero large value of results.status:

`const results = child_process_1.spawnSync(executablePath, [""]);`

I've also tried specifying explicit path to fortls.exe here and that also ended up with some large integer for results.status,

By the way, I am worrking on Windows.
